app/views/courses/manage_unlocks.html.erb

```diff
@@ -28,7 +28,23 @@
   };
 </script>
 
-<h1>Edit unlock conditions</h1>
+<h1 class="float-left">Edit unlock conditions</h1>
+
+<div class="float-right">
+  <p>Syntax:</p>
+<pre class="unlock-help">
+exercise [group] &lt;exercise-or-group&gt;
+point[s] &lt;list-of-point-names&gt;
+&lt;N&gt;% [in|of|from] &lt;exercise-or-group&gt;
```

Comment:
Are the in|of|from words really optional? If not, maybe remove the angle brackets around them